CHANGELOG.md

```diff
@@ -1,17 +1,21 @@
 # Cromwell Change Log
 
-## 0.20
-
-* The default per-upload bytes size for GCS is now the minumum 256K
-instead of 64M. There is also an undocumented config key
-`google.upload-buffer-bytes` that allows adjusting this internal value.
-
-* Updated Docker Hub hash retriever to parse json with [custom media
-types](https://github.com/docker/distribution/blob/05b0ab0/docs/spec/manifest-v2-1.md).
-
-* Added a `/batch` submit endpoint that accepts a single wdl with
-multiple input files.
-
-* The `/query` endpoint now supports querying by `id`, and submitting
-parameters as a HTTP POST.
+## 0.21
 
+* Add support for Google Private IPs through `noAddress` runtime attribute. If set to true, the VM will NOT be provided with a public IP address.
```

Comment:
I was not sure where to document this so I put it there.
Also not sure if we want to keep the 0.20 changelog items ?